lms/templates/js/discussion/new-post.underscore

```diff
@@ -0,0 +1,52 @@
+<form class="forum-new-post-form">
+    <ul class="post-errors" style="display: none"></ul>
+    <div class="forum-new-post-form-wrapper"></div>
+    <% if (cohort_options) { %>
+    <div class="post-field group-selector-wrapper <% if (!is_commentable_cohorted) { print('disabled'); } %>">
+        <label class="field-label">
+            <span class="field-label-text">
+                <% //Translators: This labels the selector for which group of students can view a post %>
+                <%- gettext("Visible To:") %>
+            </span><select aria-describedby="field_help_visible_to" class="field-input js-group-select" name="group_id" <% if (!is_commentable_cohorted) { %>disabled<% } %>>
```

Comment:
Above you changed this to use a single code block with the print function. I am fine with either, but it should be consistent.